### PR TITLE
[Dev-11261] rename data act broker to data accountability broker

### PR DESCRIPTION
**Pull request by sukhusaspending**
**Description:**
High level description of what the PR addresses should be put here. Should be detailed enough to communicate to a PO what this PR addresses without diving into the technical nuances

**Technical details:**
The technical details for the knowledge of other developers. Any detailed caveats or specific deployment steps should be outlined here.

**Requirements for PR merge:**

1. [X] Unit & integration tests updated
2. [X] API documentation updated
3. [X] Necessary PR reviewers:
    - [x] Backend
    - [ ] Frontend <OPTIONAL>
    - [ ] Operations <OPTIONAL>
    - [ ] Domain Expert <OPTIONAL>
8. [x] Jira Ticket [DEV-11261](https://federal-spending-transparency.atlassian.net/browse/DEV-11261):
    - [X] Link to this Pull-Request
    
**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

**Comment by sukhusaspending** [comment]
changed bunch of files to change data broker, data act broker to data accountability broker. Pls review

**Review by dpb-bah — CHANGES_REQUESTED** [review]
Hey @sukhusaspending,  I think the ticket is poorly worded since it's just a copy of what we did on the Broker application. The name brand changes are
* "DATA Act Broker Submission" (DABS) => "Data Accountability Broker Submission"
* "The DATA Act Broker"/ "The Data Broker" => "Data Broker"

So a lot of these are not the Data Accountability Broker Submission scenarios. Just anything involving "DATA Act Broker Submission"s should be updated to "Data Accountability Broker Submission". Otherwise, just make sure the other cases of the Data Act Broker becomes simply "Data Broker".

**Comment by sukhusaspending** [comment]
Hi Daniel, please review again based on new AC

**Review by dpb-bah — DISMISSED** [review]

**Review by collinwr — CHANGES_REQUESTED** [review]
I'm not sure updating old migrations it the best path forward here. Is it possible to create new migrations to update these models?

File: usaspending_api/financial_activities/migrations/0001_initial.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='FinancialAccountsByProgramActivityObjectClass',
             fields=[
-                ('data_source', models.TextField(choices=[('USA', 'USAspending'), ('DBR', 'DATA Act Broker')], help_text='The source of this entry, either Data Broker (DBR) or USASpending (USA)', null=True)),
+                ('data_source', models.TextField(choices=[('USA', 'USAspending'), ('DBR', 'Data Broker')], help_text='The source of this entry, either Data Broker (DBR) or USASpending (USA)', null=True)),
```
Comment (aguest-kc) [inline usaspending_api/financial_activities/migrations/0001_initial.py:24]: This TextField needs to be updated in a separate migration.

File: usaspending_api/financial_activities/migrations/0001_initial.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ class Migration(migrations.Migration):
             name='TasProgramActivityObjectClassQuarterly',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('data_source', models.TextField(choices=[('USA', 'USAspending'), ('DBR', 'DATA Act Broker')], help_text='The source of this entry, either Data Broker (DBR) or USASpending (USA)', null=True)),
+                ('data_source', models.TextField(choices=[('USA', 'USAspending'), ('DBR', 'Data Broker')], help_text='The source of this entry, either Data Broker (DBR) or USASpending (USA)', null=True)),
```
Comment (aguest-kc) [inline usaspending_api/financial_activities/migrations/0001_initial.py:81]: This TextField needs to be updated in a separate migration.

**Review by aguest-kc — CHANGES_REQUESTED** [review]

File: usaspending_api/financial_activities/migrations/0001_initial.py (hunk 1 above)
Comment (ayubshahab) [reply usaspending_api/financial_activities/migrations/0001_initial.py]: Looks like we forgot this one, fixed it.

File: usaspending_api/financial_activities/migrations/0001_initial.py (hunk 2 above)
Comment (ayubshahab) [reply usaspending_api/financial_activities/migrations/0001_initial.py]: same as above

**Review by ayubshahab — APPROVED** [review]

**Review by collinwr — APPROVED** [review]

**Pull request merged**